ts/parsers/colorparser.ts

```diff
@@ -17,6 +17,9 @@ export default class ColorParser {
 	}
 	public static getShapeFill(element): PowerpointElement["shape"]["fill"] {
 		//spPR takes precdence
+		if(!element["p:spPr"]) {
```

Comment:
```suggestion
		if (!element["p:spPr"]) {
```

Reply:
could empty be a valid value for this? in other words, can we use a better is set or is empty check?

Reply:
using loadash isEmpty now. will change it everywhere else also wherever i notice and feel will be better to use.